### ISSUE TITLE
Keep getting "Jobstore not created yet, toil job may be finished or just starting", doesn't seem to change

### ISSUE THREAD

**Issue by allanbolipata** [mskcc/ridgeback#12]
```
[2019-12-04 23:31:56,545: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:56,588: WARNING/ForkPoolWorker-15] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:56,791: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:56,844: WARNING/ForkPoolWorker-15] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:57,020: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:57,102: WARNING/ForkPoolWorker-15] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:57,312: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:57,398: WARNING/ForkPoolWorker-15] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:57,586: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:57,676: WARNING/ForkPoolWorker-15] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:57,830: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:57,955: WARNING/ForkPoolWorker-15] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:58,054: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:58,214: WARNING/ForkPoolWorker-15] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:58,289: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:58,461: WARNING/ForkPoolWorker-15] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:58,553: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:58,723: WARNING/ForkPoolWorker-15] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:58,804: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:58,977: WARNING/ForkPoolWorker-15] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:59,041: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:59,263: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:59,266: WARNING/ForkPoolWorker-15] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:59,474: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:59,517: WARNING/ForkPoolWorker-15] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:59,702: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:31:59,799: WARNING/ForkPoolWorker-15] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:32:00,174: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:32:00,344: WARNING/ForkPoolWorker-15] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:32:00,424: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:32:00,672: INFO/ForkPoolWorker-15] Task toil_orchestrator.tasks.check_status_of_command_line_jobs[19a7c477-1777-4f8e-ac1b-ddee69037713] succeeded in 18.282900581136346s: None
[2019-12-04 23:32:00,703: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:32:00,981: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:32:01,235: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:32:01,496: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:32:01,807: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:32:02,081: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:32:02,366: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:32:02,386: INFO/MainProcess] Received task: toil_orchestrator.tasks.check_status_of_command_line_jobs[a1c25c33-e66f-47e7-9bb4-efb90bb66c8a]
[2019-12-04 23:32:02,438: WARNING/ForkPoolWorker-19] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:32:02,612: WARNING/ForkPoolWorker-16] Jobstore not created yet, toil job may be finished or just starting
[2019-12-04 23:32:02,722: WARNING/ForkPoolWorker-19] Jobstore not created yet, toil job may be finished or just starting
```

**Comment by allanbolipata** [comment]
Related to #14, should be resolved together

**Comment by allanbolipata** [comment]
This was fixed once `toil` was updated to `3.21.0`.
